packages/app/src/cli/services/dev/processes/setup-dev-processes.ts

```diff
@@ -150,6 +162,9 @@ async function setPortsAndAddProxyProcess(processes: DevProcesses, proxyPort: nu
         const targetPort = await getAvailableTCPPort()
         rules[process.options.pathPrefix] = `http://localhost:${targetPort}`
         process.options.port = targetPort
+      } else if (process.type === 'graphiql') {
+        const targetPort = process.options.port
+        rules[process.prefix] = `http://localhost:${targetPort}`
```

Comment:
Given that we have a prefix `extensions` [here](https://github.com/Shopify/cli/blob/1e398864aa427179a8545ee7e9dc7821ddd125de/packages/app/src/cli/services/dev/processes/previewable-extension.ts#L81), my suspicion is that `graphiql` is the least of our problems.

Maybe we can discuss within the team, align on a convention, and update everything.

Reply:
Note for posterity: We decided to add the namespace, without backporting to `extensions`. Maybe we'll do that one day, too, in case developers have their own `/extensions` route 🤷